### DISCUSSION TITLE
[科普]差分宇宙的Boss萨姆不能用来做安静燃烧成就

### DISCUSSION THREAD

**Discussion by Dave-12138** [share121/inter-knot#1717] [崩坏：星穹铁道]
![image](https://github.com/user-attachments/assets/ec385761-c5bb-45e6-9f88-6b2a00938cd0)

因为她不叫 **「星核猎手」萨姆** ，她的名字是 **「星核猎手」萨姆（完整）** 

![image](https://github.com/user-attachments/assets/29876a2b-369b-4e92-a65a-ccc3ee3d6c8e)

**Comment by Ste11er** [comment]
这

**Comment by whsjjj28288** [comment]
虽然我没有抽到过流莹，但是为此大受震撼

**Comment by Dave-12138** [comment]
> 虽然我没有抽到过流莹，但是为此大受震撼

没抽也没关系，只要你还没完成安静燃烧成就，这周去差分宇宙都能震撼（

**Comment by GanlinSama** [comment]
天才
